phoenix-shared/src/commonMain/kotlin/fr.acinq.phoenix/utils/MnemonicLanguage.kt

```diff
@@ -0,0 +1,83 @@
+package fr.acinq.phoenix.utils
+
+import fr.acinq.bitcoin.MnemonicCode
+
+enum class MnemonicLanguage {
+    English {
+        override val code: String get() = "en"
+        override fun wordlist(): List<String> {
+            return MnemonicCode.englishWordlist
+        }
+        override fun matchMap(): Map<String, String> {
+            return mapOf()
+        }
+    },
+    Spanish {
+        override val code: String get() = "es"
+        override fun wordlist(): List<String> {
```

Comment:
To be consistent those word lists should be added to bitcoin-kmp in https://github.com/ACINQ/bitcoin-kmp/blob/aa5dc68e25700df53f92f4c31d7f1c0be42fcf8c/src/commonMain/kotlin/fr/acinq/bitcoin/MnemonicCode.kt#L22

Reply:
We can do that. Do we only want to add the word lists to bitcoin-kmp? Or do you think it would be helpful if we added the entire `MnemonicLanguage.kt`?

Reply:
Let's keep the additional word lists here for now.